README.md

```diff
@@ -90,13 +97,31 @@ You can use a [PAT](https://help.github.com/en/github/authenticating-to-github/c
 
 ```yml
       - name: Slash Command Dispatch
-        uses: peter-evans/slash-command-dispatch@v1
+        uses: peter-evans/slash-command-dispatch@v2
         with:
-          token: ${{ secrets.REPO_ACCESS_TOKEN }}
-          reaction-token: ${{ secrets.REPO_ACCESS_TOKEN }}
-          commands: deploy, integration-test, build-docs
+          token: ${{ secrets.PAT }}
+          reaction-token: ${{ secrets.PAT }}
+          commands: |
+            deploy
+            integration-test
+            build-docs
 ```
 
+#### `dispatch-type`
+
+By default, the action creates [repository_dispatch](https://developer.github.com/v3/repos/#create-a-repository-dispatch-event) events.
+Setting `dispatch-type` to `workflow` will instead create [workflow_dispatch](https://docs.github.com/en/actions/reference/events-that-trigger-workflows#workflow_dispatch) events.
+There are significant differences in the action's behaviour when using `workflow` dispatch. See [workflow dispatch](docs/workflow-dispatch.md) for usage details.
+
+For the majority of use cases, the default `repository` dispatch will likely be the most suitable for new workflows.
+If you already have `workflow_dispatch` workflows, you can execute them with slash commands using this action.
+
+| Repository Dispatch (default) | Workflow Dispatch |
+| --- | --- |
+| Events are created with a `client_payload` giving the target workflow access to a wealth of useful [context properties](https://github.com/peter-evans/slash-command-dispatch/tree/dev#accessing-contexts). | A `client_payload` cannot be sent with [workflow_dispatch](https://docs.github.com/en/actions/reference/events-that-trigger-workflows#workflow_dispatch) events. The target workflow can only make use of up to 10 pre-defined inputs, the names of which must match named arguments supplied with the slash command. |
+| Slash commands can only execute workflows in the target repository's default branch. | Slash commands can execute workflows in any branch using the `ref` named argument. The reference can be a branch, tag, or a commit SHA. |
```

Comment:
> Slash commands can only execute workflows in the target repository's default branch.

this sounds like that this type would not be usefull in PRs, as I might assume while reading that I cannot run slash commands on my PRs branch..